index.html

```diff
@@ -25,6 +25,13 @@ <h1>Bookbinder JS</h1>
             <center style="font-family:monospace;
     padding-top: 0.75em;font-variant-caps: small-caps;">current version: %PACKAGE_VERSION%</center>
         </div>
+        <div class="section" id="settings">
+            <h2>Settings</h2>
+            <span class="row">
+                <button type="button" name="reset_settings" id="reset_settings">Reset</button>
+                <span data-balloon-length="medium" aria-label="The imposer is able to save and load your current settings via the site URL. Clicking this button will reset those to the default settings." data-balloon-pos="up">ℹ️</span>
```

Comment:
Final nit-- happy to have a `Settings` section! Happy to have a `reset` button! Could the info in the mouse-over (i) be pulled out into just plain explanatory text however? I think it'd make the lonely little button look a bit better - we don't really need to save screen real-estate there atm